### PR TITLE
Add DivvyCloud action to workflows

### PR DESCRIPTION
**Pull request by GaryGSC**

### CODE REVIEW

**Comment by GaryGSC** [comment]
This isn't merge-able until we've added the DivvyCloud secrets to this repo, hence the draft PR.

**Review by rayos006 — APPROVED** [review]

**Comment by GaryGSC** [comment]
The code changes LGTM now. The only remaining reservation I have is that if we want people across campus to use this template, how are they going to get the `DIVVYCLOUD_USERNAME` and `DIVVYCLOUD_PASSWORD` secrets?

Should we add a note to the README to contact the Cloud Office?

**Comment by GaryGSC** [comment]
BTW, @rayos006, we're getting some weird output because of what we're getting from the API.

We have [some `null` resources](https://github.com/byu-oit/hw-fargate-api/runs/1427375650#step:8:960)
```
▶ Access List Exposes SSH to World (Security Group)
  • module.app.module.my_fargate_api.aws_security_group.alb-sg
  • module.app.module.my_fargate_api.aws_security_group.fargate_service_sg
  • null
  • null
```
Because [the DivvyCloud API says there are `null` `address`es for some resources](https://github.com/byu-oit/hw-fargate-api/runs/1427375650#step:8:791)
```
"serverlessfunction:9461": {
  "address": null,
  "id": 9461,
  "name": "hw-fargate-api-deploy-test-dev-postman-tests",
  "provider_id": "hw-fargate-api-deploy-test-dev-postman-tests",
  "region": "us-west-2",
  "type": "serverlessfunction"
}
```

**Review by joshgubler — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -181,7 +181,7 @@ jobs:
           terraform-plan-file: plan
 
       - name: Analyze Terraform Plan
-        uses: byu-oit/github-action-tf-plan-analyzer@v2
+        uses: byu-oit/github-action-tf-plan-analyzer@v2.0.1
```
Comment (GaryGSC) [inline .github/workflows/ci.yml:184]: 👀 I'd like to think we're effectively treating `v2` as `^2.0.0`, so this changes us from `^2.0.0` to `2.0.1` (without the `^`).

Comment (joshgubler) [reply .github/workflows/ci.yml:184]: So per [this conversation in Slack](https://byu-oit.slack.com/archives/CQ2BE663T/p1605907223154800), we should change this back to `v2`, and update `github-action-tf-plan-analyzer` to use `v2` as a branch that gets updated with minor/patch changes.

**Review by joshgubler — APPROVED** [review]

**Comment by GaryGSC** [comment]
This LGTM now, so I'm going to go ahead and merge it.

**Pull request merged**